include/ica_api.h

```diff
@@ -3317,6 +3317,12 @@ kma_ctx* ica_aes_gcm_kma_ctx_new();
  * @param iv
  * Pointer to a readable buffer of size greater than or equal to iv_length
  * bytes, that contains an initialization vector of size iv_length.
+ * When running in fips mode, the pointer may alternatively be NULL, in which
```

Comment:
Well, this sentence is not 100% true:
_**When running in fips mode**, the pointer may alternatively be NULL_
Actually, the pointer can also be NULL when not in FIPS mode.....
Maybe rephrase this that for FIPS approved operation this pointer MUST be NULL or similar.

Reply:
Good catch: yes, I initially had this only for fips, but then decided to allow an internal iv also in non-fips mode. This gives applications more flexibility. But I didn't update the comment. Will do ...